presto-cassandra/src/test/java/io/prestosql/plugin/cassandra/CassandraTestingUtils.java

```diff
@@ -247,4 +249,36 @@ private static void insertTestData(CassandraSession session, SchemaTableName tab
         }
         assertEquals(session.execute("SELECT COUNT(*) FROM " + table).all().get(0).getLong(0), rowsCount);
     }
+
+    private static void createTableUserDefinedType(CassandraSession session, SchemaTableName table)
+    {
+        String typeName = "type_user_defined";
+
+        session.execute("DROP TABLE IF EXISTS " + table);
+        session.execute("DROP TYPE IF EXISTS " + table.getSchemaName() + "." + typeName);
+
+        session.execute("CREATE TYPE " + table.getSchemaName() + "." + typeName + " (" +
```

Comment:
please use `String.format` instead of concatenation (unless you are adding something at the end like `("string: " + var)`